src/meteors/lime.py

```diff
@@ -1605,16 +1636,25 @@ def get_spectral_attributes(
                 If equals to None, the band mask is created within the function. Defaults to None.
             target (int, optional): If the model creates more than one output, it analyzes the given target.
                 Defaults to None.
-            band_names (list[str] | dict[str | tuple[str, ...], int] | None, optional): Band names. Defaults to None.
+            n_samples (int, optional): The number of samples to generate/analyze in LIME. The more the better but slower. Defaults to 10.
+            perturbations_per_eval (int, optional): The number of perturbations to evaluate at once (Simply the inner batch size).
+                Defaults to 4.
             verbose (bool, optional): Specifies whether to show progress during the attribution process. Defaults to False.
+            model_segmentation_postprocessing_for_segmentation_problem_type: (Callable[[torch.Tensor, torch.Tensor], torch.Tensor] | None):
```

Comment:
`model_segmentation_postprocessing_for_segmentation_problem_type` seems a bit too long, would have changed it into something like `postprocess_segmentation_output_to_1d`